scout/constants/gene_tags.py

```diff
@@ -21,6 +23,8 @@
     "AD (imprinting)": {"bgcolor": "bg-cyan", "text_color": "text-dark"},
     "AR": {"bgcolor": "bg-pink", "text_color": "text-white"},
     "XL": {"bgcolor": "bg-purple", "text_color": "text-white"},
+    "Y": {"bgcolor": "bg-purple", "text_color": "text-white"},
+    "MT": {"bgcolor": "bg-purple", "text_color": "text-white"},
```

Comment:
I wonder if I need to add something also to [this dictionary](https://github.com/Clinical-Genomics/scout/blob/fec5cd94f5cb800b0bc213072d90c78b9ab080af/scout/constants/variant_tags.py#L57) @dnil ?

Reply:
That would be if we use it to color them on the variant page? Lets see...

Reply:
They are not (yet 😉) in color on the gene panel page.

Reply:
Feel free to push this PR if you want!

Reply:
Yep, they will be used if you add them there, but written as a plain info badge otherwise.

Reply:
Perhaps it is fine then. I'll test if it shows on the variant page if you provide free text strings. I suspect it doesn't